jenkins/run

```diff
@@ -53,15 +53,36 @@ fi
 
 _image=${IMAGE:-${_image_repo}/pbench-${_image_role}-${_image_kind}:${_branch_name}}
 
-podman run \
+if [ -n "${CONTAINER_HOST}" ]; then
+    # Since CONTAINER_HOST is defined, when we want to run another container, do
+    # it with `podman-remote` (which will run it where CONTAINER_HOST points).
+    # This works nicely, e.g., if we are already inside a properly configured
+    # container.
+    PODMAN="podman-remote"
+else
+    # Since CONTAINER_HOST is not defined, don't invoke the container remotely.
+    # However, we might want to run a container "remotely" on this host;
+    # therefore, enable the service which makes it available to receive
+    # `podman-remote` invocations.  This creates a listener on
+    # /run/user/$(id -u)/podman/podman.sock, which will be mapped into the
+    # container in the invocation below so that we can create peer-containers
+    # on this host from inside the container.
+    PODMAN="podman"
+    systemctl --user enable --quiet --now podman.socket
+fi
+
+${PODMAN} run \
     --userns=keep-id \
+    --security-opt label=disable \
     --volume ${HOME_DIR} \
     --volume ${HOME_DIR}/.config \
-    --volume $(pwd):${HOME_DIR}/pbench:z \
+    --volume $(pwd):${HOME_DIR}/pbench \
```

Comment:
I notice you're explicitly excluding `selinux` *updates*, and presumably the target sidecar container won't have `selinux` started. I guess that means the `:z` labeling suffix is meaningless??

Reply:
Whether or not `selinux` is or should be started seems to be a mystery.  However, the container invocation disables it (in no small part because I believe we don't need it), and it doesn't seem to be there when things run.  As a result, the `:z` directive has no effect, so I dropped it.